tests/fpga/long_long_opencl_test.py

```diff
@@ -1,19 +1,13 @@
-## copied jacobi 1d and simplified the kernel
+## simple test case to check that for the intel fpga the openCL type long gets used instead of C type long long
```

Comment:
Starts with capital letter (and use multi line comments with `'''` if it does not fit in a line with 120 chars)